nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -70,7 +107,7 @@ def __len__(self):
 
 
 @experimental
-class AudioToCharDALIDataset(Iterator):
+class _AudioTextDALIDataset(Iterator):
```

Comment:
Is there a way to merge _AudioTextDALIDataset and _AudioTextDataset and use _AudioTextDataset for dali? or they are that much different?

Reply:
Not possible. Enormously different due to Dali pipeline vs Nemo processing